README.md

```diff
@@ -1,24 +1,16 @@
-**[DISCLAIMER]**
--------------------
-
-The "Blaze" Sidechain SDK development is in progress, and the release is coming soon!
-
-The version on master is outdated. The most recent version can be found in the "dev" branch, and can be tested by early adopters on our official testnet.
-
-Please note: the current version is experimental and a work in progress. Use at your own risk.
-
-**Sidechains-SDK Beta**
+**Sidechains-SDK Blaze**
 -------------------
 Sidechains are an innovation devised to enable blockchain scalability and extensibility by creating parallel platform and application layers that are bound to the mainchain without imposing a significant burden. Each sidechain implements the desired features and custom business logic, rooted in a protocol that offers a way to transfer coins from and to the original mainchain and each sidechain.
 
 Zendoo is a unique sidechain and scaling solution developed by Horizen. The Zendoo sidechain SDK is a framework that supports the creation of sidechains and their custom business logic, with the Horizen public blockchain as the mainchain. A detailed description of the concept, the protocol, and details about mainchain/sidechain interaction can be found in the [Zendoo Whitepaper](https://www.horizen.global/assets/files/Horizen-Sidechain-Zendoo-A_zk-SNARK-Verifiable-Cross-Chain-Transfer-Protocol.pdf).
 
-**Beta Features**
+**Blaze Features**
 
-* The Cross-Chain Transfer Protocol (CCTP) implementation to support sidechain declaration, forward transfers, backward transfer requests, and withdrawal certificates
+* The Cross-Chain Transfer Protocol (CCTP) implementation to support sidechain declaration, forward transfers, backward transfer requests, withdrawal certificates and ceased sidechain withdrawals
 * Basic zk-SNARK threshold signature verification circuit to authenticate withdrawal certificates See [zendoo-sc-cryptolib](https://github.com/HorizenOfficial/zendoo-sc-cryptolib)
```

Comment:
A dot ('.') is missing before "See [zendoo-sc-cryptolib]"

Reply:
Done in https://github.com/HorizenOfficial/Sidechains-SDK/pull/338